CONTRIBUTING.rst

```diff
@@ -55,10 +57,10 @@ To set up `oemof-solph` for local development:
 
     tox
 
-5. Commit your changes and push your branch to GitHub::
+5. Commit your changes and push your branch to GitHub. Please do not forget to write a descriptive commit message that eventually explains design decisions::
```

Comment:
From personal preference, I would remove the "eventually" in the added sentence.

Reply:
```suggestion
5. Commit your changes and push your branch to GitHub. Please do not forget to write a descriptive commit message that explains design decisions::
```